app/templates/doc_pages/_headings_toc.html.erb

```diff
@@ -1,12 +1,12 @@
-<ol data-testid="headings-toc">
+<%= tag.ol("data-testid" => testid) do %>
```

Comment:
This here was me making sure we don't have multiple "data-testid=headings-toc" elements on the page, which is necessary for the newly added test to work.